docs/products/kafka/howto/kafka-sasl-auth.rst

```diff
@@ -1,18 +1,22 @@
 Use SASL Authentication with Apache Kafka®
```

Comment:
- Consider not capitalizing `authentication`.
- Consider adding `Aiven for` before `Apache Kafka`

Proposal: `Use SASL authentication with Aiven for Apache Kafka®`

Reply:
Fixed